rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.h

```diff
@@ -21,7 +21,7 @@ namespace rsx
 			label m_body{};
 			label m_titles{};
 
-			bool m_framerate_graph_enabled{};
+			bool m_fps_graph_enabled{};
```

Comment:
why the rephrase

Reply:
for consistency. the fps graph object is called `m_fps_graph`
the framerate enable bool + graph are called the same, so why not that too

Reply:
anyway I see it's called framerate in the rest of the code you've added so I'll revert